homeassistant/components/homekit/__init__.py

```diff
@@ -389,6 +392,8 @@ def start(self, *args):
 
         _LOGGER.debug("Driver start")
         self.hass.add_job(self.driver.start)
+        self.hass.bus.async_fire(EVENT_HOMEKIT_STARTED)
```

Comment:
Why still fire the events ?

Reply:
@balloob i was using it to get a notification when homekit started since homekit is very tied to my WAF haha. i felt homekit is kinda a blackbox that doesn't really have a status indicator anywhere beyond just something showing up in logs if it errors.

Reply:
You can add logging to make it less of a blackbox. We should not include events.

Reply:
@balloob ok ive limited this pull to logbook.